index.html

```diff
@@ -14,172 +14,200 @@
     <div class="wrapper">
         <header class="nav-bar">
             <a href="#">
-                <p>Flex Box</p>
+                <p>Hamma Azarok</p>
             </a>
             <nav> <img src="img/Enabled.png" alt="Logo"></nav>
         </header>
-        <section class="header">
-            <h1 class="header-text">I’m Hamma
-                Glad to see you!</h1>
-            <p>
-                I’m a software developer! I can help you build a product , feature or website Look through some of my
-                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-                contact me.
-            </p>
-            <a href="#">LET’S CONNECT</a>
-            <ul class="social-links">
-                <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
-                <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
-                <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
-                <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
-                <li> <a href="#"><img src="img/icons/Vector3.png" alt=""></a></li>
+        <nav class="nav-bar-desktop">
+            <a class="logo" href="#">Hamma Azarok</a>
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#portfolio">Portofolio</a></li>
+                <li class="nav-item"><a href="#about">About</a></li>
+                <li class="nav-item"><a href="#contact">Contact</a></li>
             </ul>
+        </nav>
+        <section class="header">
+            <div class="header-content">
+                <h1 class="header-text">I’m Hamma
+                    Glad to see you!</h1>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
```

Comment:
- [ ] Your portfolio should be responsive on all screen devices right now but in 700px wide screen it is not fully responsive please try to remove the white space and also should all the content of the portfolio responsive 
![image](https://user-images.githubusercontent.com/61790587/171682598-c0b011f9-5621-4e2a-9410-b4b6ac38730b.png)